store/sqlstore/post_store.go

```diff
@@ -1327,7 +1327,6 @@ func (s *SqlPostStore) GetParentsForExportAfter(limit int, afterId string) store
                     AND p1.DeleteAt = 0
 					AND Channels.DeleteAt = 0
 					AND Teams.DeleteAt = 0
-					AND Users.DeleteAt = 0
```

Comment:
You need to make the same change as here in `GetRepliesForExport` as well, as otherwise it will only include "root" posts with deleted users in the export.

Reply:
@grundleborg Thanks for review!
Pushed this change.